webclient/heslar/migrations/0003_default.py

```diff
@@ -8,7 +8,7 @@ class Migration(migrations.Migration):
     initial = True
 
     dependencies = [
-        ('heslar', '0002_initial'),
+        ('heslar', '0003_auto_20230327_1318'),
```

Comment:
@pesikj mnel jsi dependency na to same a nechtelo to projit cez testy, tak jsem to upravil.

Reply:
Ok.